roles/sushy_emulator/defaults/main.yml

```diff
@@ -18,17 +18,33 @@
 # All variables intended for modification should be placed in this file.
 # All variables within this role should have a prefix of "cifmw_sushy_emulator"
 
+cifmw_sushy_emulator_basedir: "{{ ansible_user_dir ~ '/ci-framework-data' }}"
+cifmw_sushy_emulator_container_name: "cifmw-sushy_emulator"
 cifmw_sushy_emulator_driver: libvirt
-cifmw_sushy_emulator_sshkey_path: "{{ ansible_user_dir }}/.ssh/id_cifw"
-cifmw_sushy_emulator_libvirt_user: zuul
-cifmw_sushy_emulator_listen_ip: 0.0.0.0
 cifmw_sushy_emulator_driver_openstack_client_config_file: /etc/openstack/clouds.yaml
 cifmw_sushy_emulator_driver_openstack_cloud: None
-cifmw_sushy_emulator_namespace: sushy-emulator
-cifmw_sushy_emulator_redfish_username: admin
-cifmw_sushy_emulator_redfish_password: password
-cifmw_sushy_emulator_resource_directory: "{{ (ansible_user_dir, 'ci-framework-data', 'artifacts', 'sushy_emulator') | path_join }}"
-cifmw_sushy_emulator_image: quay.io/metal3-io/sushy-tools:latest
-cifmw_sushy_emulator_instance_node_name_prefix: cifmw-
-cifmw_sushy_emulator_container_name: "cifmw-sushy_emulator"
 cifmw_sushy_emulator_install_type: 'ocp'
+cifmw_sushy_emulator_image: quay.io/metal3-io/sushy-tools:latest
+cifmw_sushy_emulator_ipmi_uri: >-
```

Comment:
This variable name is strange.
Why `ipmi` ? Should it not be `cifmw_sushy_emulator_libvirt_uri` ?